### ISSUE TITLE
resolvePath strips leading slash which breaks writes on Android

### ISSUE THREAD

**Issue by wodCZ** [leethree/redux-persist-fs-storage#1]
[These lines](https://github.com/leethree/redux-persist-fs-storage/blob/master/index.js#L7-L12) strip leading slash from `location` (eg /data/data/com.myapp/cache), which then causes write error:

```
Error: ENOENT: no such file or directory, open 'data/data/com.myapp/cache/myFolder/somekey'
```

This happens because of `.filter(part => part && part !== '.')`

In my opinion, `resolvePath` should always return absolute path with leading `/`.

Simple prepending of `/` to function result fixes this issue for me - I can prepare Pull Request if you want.

Thanks!

**Comment by 7ynk3r** [comment]
same here. is this happening only in the lastest?

**Comment by 7ynk3r** [comment]
@wodCZ can you put your PR?

**Comment by wodCZ** [comment]
@7ynk3r done ⬆️

**Comment by 7ynk3r** [comment]
@leethree can you take a look at https://github.com/leethree/redux-persist-fs-storage/pull/2? so we can close this issue and move on

**Comment by leethree** [comment]
Sorry for the delay. Thanks for fixing the issue!

**Comment by leethree** [comment]
Should be fixed in v1.2.1.

**Comment by 7ynk3r** [comment]
@leethree I can't wait for the next release! 
@wodCZ thanks for fixing this!
